src/skeleton/mod.rs

```diff
@@ -346,6 +346,17 @@ fn ignore_all_members_except(manifests: &mut [ParsedManifest], member: String) {
         .and_then(|toml| toml.contents.get_mut("workspace"))
         .and_then(|workspace| workspace.get_mut("members"))
     {
-        *members = toml::Value::Array(vec![toml::Value::String(member)]);
+        let mut search = member.clone();
+        search.insert_str(0, "/");
+        match members {
+            cargo_manifest::Value::Array(arr) => arr.retain(|i| {
+                if let cargo_manifest::Value::String(item) = i {
+                    item.contains(&search) || item.eq(&member)
```

Comment:
This feels a bit too crude. 
We should be analyzing the existing `Cargo.toml`, build a `Path <> binary name` map and then use that here.